desc/equilibrium/equilibrium.py

```diff
@@ -461,10 +519,10 @@ def __repr__(self):
             type(self).__name__
             + " at "
             + str(hex(id(self)))
-            + " (L={}, M={}, N={}, NFP={}, sym={}, spectral_indexing={})".format(
-                self.L, self.M, self.N, self.NFP, self.sym, self.spectral_indexing
-            )
-        )
+            + f" (L={self.L}, M={self.M}, N={self.N}, Lz={self.Lz}, Mz={self.Mz},"
+            + "Nz={self.Nz}, NFP={self.NFP}, sym={self.sym},"
+            + "spectral_indexing={self.spectral_indexing})"
+        )  # RG: not sure if this change is needed
```

Comment:
we should track the omega resolution so I think yes it is not needed strictly but we want this to be changed